views/partials/body.handlebars

```diff
@@ -57,7 +57,7 @@
 
             <div> <!--container buttons and limit repo message -->
                   <button id="btn-toggle" class="funbtn active" onclick="toggleColor()">
-                    <i class="fa fa-toggle-on" aria-hidden="true"></i>
```

Comment:
Hi, I tried this on my local but the result is there's no toggle-on button because you delete this line, is this correct? @fuzztone313

Reply:
Yes, I deleted because I'm using ::before pseudo on button element wtih unicode fontAwesome in "content: ''\unicode fontawesome" css property.

Reply:
Okay I got that It's now resolved :)